x/ibc/04-channel/client/cli/query.go

```diff
@@ -0,0 +1,75 @@
+package cli
```

Comment:
Need to add ability to query which `channels` exist on a given chain and/or connection.